Sources/ApplicativeRouter/Combinators.swift

```diff
@@ -75,22 +75,22 @@ public let dataBody = Router<Data>(
 public let stringBody = dataBody.map(PartialIso.data.inverted)
 
 /// Processes the body data of the request into form data of (key, value) pairs.
-public let formDataBody = stringBody.map(.formEncodedFields)
+public let formEncodedBodyFields = stringBody.map(.formEncodedFields)
```

Comment:
this is just me renaming... while debugging I got confused what this actually was cause the name was a bit ambiguous. open to other names

Reply:
`formEncodedDataBody` for consistency?